config/initializers/fast_gettext.rb

```diff
@@ -31,5 +31,5 @@ def available_locales
 
 FastGettext.default_text_domain       = 'app'
 
-I18n.default_locale        = available_locales.for(:i18n).first
-FastGettext.default_locale = available_locales.for(:fast_gettext).first
+I18n.default_locale        = default_locale.gsub(/_/, '-')
+FastGettext.default_locale = default_locale.gsub(/\-/, '_')
```

Comment:
I think it would be cleaner to use the new LocaleFormatter class here:
https://github.com/DMPRoadmap/roadmap/blob/development/lib/locale_formatter.rb

```
I18n.default_locale = LocaleFormatter.new(default_locale, format: :i18n)
FastGettext.default_locale = LocaleFormatter.new(default_locale, format: :fast_gettext)
```